views/signup.ejs

```diff
@@ -3,11 +3,11 @@
     <p>
```

Comment:
Maybe we can use the same view for both login and sign up? They're practically the same except for the title and the form action

Reply:
Would we use a variable for the form action? I think this hurts us to scale the sign up in the future. We would likely want more info in the future and also incorporate the FB and google OAUTH down the road.

Reply:
I accept :)